app/src/main/java/com/rockthevote/grommet/ui/eventFlow/SessionTimeTrackingViewModel.kt

```diff
@@ -30,8 +35,14 @@ class SessionTimeTrackingViewModel(
     private val _effect = LiveEvent<SessionSummaryState.Effect?>()
     val effect: LiveData<SessionSummaryState.Effect?> = _effect
 
+    private val _clockOutData = LiveEvent<ClockInOutTimes>()
```

Comment:
the session summary needs more than just clock in/out data and we're already passing in clock-in time in the session status object , so why create this new data class?

Reply:
It's not for the session. The session is invalidated and nullified as soon as the user taps "Clock Out". We need something to persist so we can display the clock in/out time after the session is wiped.